src/model/json-schema.json

```diff
@@ -2490,7 +2490,7 @@
                 "type"
             ]
         },
-        "Pillar": {
+        "CAPIPillar": {
```

Comment:
Should this change be a part of this PR? It looks like a breaking change if not matched with Frontend cc/ @oliverlloyd

Reply:
Not sure how this came about to be honest but I suspect if you merge in `main` it should go away?

Reply:
Yes, Ive merged main and its gone away.